staff/templates/staff.html

```diff
@@ -48,7 +44,8 @@ <h2>{{ form.form.name }}</h2><hr>
                 <p><a href="{{ form.form.esign.template }}">Esign Here</a></p><br>
               {% endif %}
               <p style="padding-bottom: 10px;">
-                  <input type="button" class="mui-btn" value="Download Form" onclick="window.location.href='{{ form.form.url }}'"/>
+                  <p>URL: {{ form.form.form }}</p>
+                  <input type="button" class="mui-btn" value="Download Form" onclick="window.location.href='{{ form.form.form }}'"/>
```

Comment:
This should be a link, no javascript.

Reply:
Resolved 6eb5cb979b3e7f5677a52dfe3d696b44155aa78e